### PR TITLE
Add dense LP interface

### PR DESCRIPTION
**Pull request by Bambade**
Add a setting option for solving more efficiently Linear Programs with ProxQP, as suggested by #173 (i.e., all linear algebra operations involving the Hessian cost are avoided).

This PR adds this feature for ProxQP dense backend. It will be added later for ProxQP sparse backend as well.

### CODE REVIEW

File: test/src/dense_qp_eq.cpp [hunk 1]
```diff
@@ -157,3 +157,61 @@ DOCTEST_TEST_CASE("linear problem with equality  with equality constraints and "
               << std::endl;
   }
 }
+
+DOCTEST_TEST_CASE("linear problem with equality  with equality constraints and "
+                  "linar cost and increasing dimension using wrapper API using "
```
Comment (jcarpent) [inline test/src/dense_qp_eq.cpp:162]:
```suggestion
                  "linear cost and increasing dimension using wrapper API and  "
```

File: test/src/dense_qp_eq.cpp [hunk 2]
```diff
@@ -157,3 +157,61 @@ DOCTEST_TEST_CASE("linear problem with equality  with equality constraints and "
               << std::endl;
   }
 }
+
+DOCTEST_TEST_CASE("linear problem with equality  with equality constraints and "
```
Comment (jcarpent) [inline test/src/dense_qp_eq.cpp:161]:
```suggestion
DOCTEST_TEST_CASE("linear problem with equality with equality constraints and "
```

**Review by jcarpent — APPROVED** [review]
@Bambade Could you approved my suggested change?

**Comment by costashatz** [comment]
This is great guys! Thanks! Do you have any benchmark on the performance gains?

**Comment by jcarpent** [comment]
Solving #47.

**Pull request merged**